Sources/PointFree/Database.swift

```diff
@@ -82,27 +86,15 @@ private func createUser(with envelope: GitHub.UserEnvelope) -> EitherIO<Error, P
 }
 
 private func fetchUser(with token: GitHub.AccessToken) -> EitherIO<Error, Database.User?> {
-  return execute(
+  return firstRow(
```

Comment:
A helper that's basically like Active Record's `first` using `Decodable` under the hood.